web/src/views/repo/settings/General.vue

```diff
@@ -18,22 +18,26 @@
       </InputField>
 
       <InputField
-        v-slot="{ id }"
         :label="$t('repo.settings.general.netrc_only_trusted.netrc_only_trusted')"
         docs-url="docs/usage/project-settings#custom-trusted-clone-plugins"
       >
-        <span class="mb-2 ml-1 text-wp-text-alt-100">{{ $t('repo.settings.general.netrc_only_trusted.desc') }}</span>
-
-        <div class="flex flex-col gap-2">
-          <div v-for="image in repoSettings.netrc_trusted" :key="image" class="flex gap-2">
-            <TextField :id="id" :model-value="image" disabled />
-            <Button type="button" color="gray" start-icon="trash" @click="removeImage(image)" />
-          </div>
-          <div class="flex gap-2">
-            <TextField :id="id" v-model="newImage" @keydown.enter.prevent="addNewImage" />
-            <Button type="button" color="gray" start-icon="plus" @click="addNewImage" />
+        <template #default="{ id }">
+          <div class="flex flex-col gap-2">
+            <div v-for="image in repoSettings.netrc_trusted" :key="image" class="flex gap-2">
+              <TextField :id="id" :model-value="image" disabled />
+              <Button type="button" color="gray" start-icon="trash" @click="removeImage(image)" />
+            </div>
+            <div class="flex gap-2">
+              <TextField :id="id" v-model="newImage" @keydown.enter.prevent="addNewImage" />
+              <Button type="button" color="gray" start-icon="plus" @click="addNewImage" />
+            </div>
           </div>
-        </div>
+        </template>
+        <template #description>
+          <p class="text-sm">
```

Comment:
I have refactored this part a bit more in https://github.com/woodpecker-ci/woodpecker/pull/4773 maybe we can merge this one first?